recipes/roounfold/recipe.yaml

```
@@ -0,0 +1,85 @@
+context:
+  name: RooUnfold
+  version: "3.0.5"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://gitlab.cern.ch/RooUnfold/RooUnfold/-/archive/${{ version }}/${{ name }}-${{ version }}.tar.gz
+  sha256: e57d846efcaff831dfeaad528fa9ebc60fe3e09165decbbc261592c23851d40a
+  patches:
+    - patches/cmake-minimum-3.5.patch
+
+build:
+  number: 0
+  script:
+    - cmake ${CMAKE_ARGS}  -S . -B build
+    - cmake --build build --clean-first --parallel="${CPU_COUNT}"
+    - cmake --install build
+    # NOTE: we must manually set SP_DIR since it is not set.
+    # Can be removed once https://github.com/prefix-dev/rattler-build/issues/1733 is fixed.
+    - if: unix
+      then:
+        - export SP_DIR=$(python -c "import site; print(site.getsitepackages()[0])")
+    - if: win
+      then:
+        - set "SP_DIR=%PREFIX%\\Lib\\site-packages"
+    # equivalent of the following:
+    #- rsync -rav --exclude='__pycache__/' --exclude='*.pyc' $SRC_DIR/python/ $SP_DIR/RooUnfold/
+    - python -c "import shutil, os, sysconfig; shutil.copytree(os.path.join(os.environ['SRC_DIR'], 'python'), os.path.join(os.environ['SP_DIR'], 'RooUnfold'), dirs_exist_ok=True)"
+
+requirements:
+  build:
+    - ${{ compiler("cxx") }}
+    - ${{ compiler("c") }}
+    - ${{ stdlib("c") }}
```

If you only build for `linux`, this might work:
```suggestion
    - cxx-compiler 13.*
    - c-compiler 13.*
    - sysroot_linux-64 2.17.*
```

Once the feedstock is created, you should be able to use `variants.yaml` or `conda_build_config.yaml` and revert back to the current specifications for other targets than `linux*`.